HUD/SetCurrentCharacterHudColor.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: HUD 
+---
+
+## _SET_CURRENT_CHARACTER_HUD_COLOR
+
+```c
+// 0x2ACCB195F3CCD9DE 
+void _SET_CURRENT_CHARACTER_HUD_COLOR(int hudColorId);
+```
+
```

Comment:
This native enables the script to designate a personalized color for the multiplayer HUD, akin to modifying colors for characters in multiplayer. Need to clear this by passing -1 when no longer in use.